pkg/gw/options.go

```diff
@@ -122,6 +157,11 @@ func (o Options) Bind() string {
 	return o.Address + ":" + strconv.Itoa(o.Port)
 }
 
+// MetricsBind return metrics bind address
+func (o Options) MetricsBind() string {
+	return o.Address + ":" + strconv.Itoa(o.MetricsPort)
```

Comment:
please do not format address manually , instead use Sprintf

Reply:
@volknanebo i blindly copied from `Bind()` :) I've changed it to `Sprintf` in both places.